egs/wsj/s5/steps/nnet3/lstm/train.sh

```diff
@@ -298,8 +313,9 @@ if [ $stage -le -4 ] && [ -z "$egs_dir" ]; then
   extra_opts+=(--transform-dir $transform_dir)
   extra_opts+=(--left-context $left_context)
   extra_opts+=(--right-context $right_context)
-  extra_opts+=(--valid-left-context $((chunk_width + left_context)))
-  extra_opts+=(--valid-right-context $((chunk_width + right_context)))
+  extra_opts+=(--valid-left-context $((minibatch_chunk_size + left_context)))
```

Comment:
I agree that the validation setup should match with the training condition,
but not sure of the other details of this-> I'll leave it to you.

On Mon, Jan 4, 2016 at 2:16 PM, Vijayaditya Peddinti <
notifications@github.com> wrote:

> In egs/wsj/s5/steps/nnet3/lstm/train.sh
> <https://github.com/kaldi-asr/kaldi/pull/432#discussion_r48789340>:
>
> > @@ -298,8 +313,9 @@ if [ $stage -le -4 ] && [ -z "$egs_dir" ]; then
> >    extra_opts+=(--transform-dir $transform_dir)
> >    extra_opts+=(--left-context $left_context)
> >    extra_opts+=(--right-context $right_context)
> > -  extra_opts+=(--valid-left-context $((chunk_width + left_context)))
> > -  extra_opts+=(--valid-right-context $((chunk_width + right_context)))
> > +  extra_opts+=(--valid-left-context $((minibatch_chunk_size + left_context)))
>
> Keep this as chunk-width. I think that might lead to a better match with
> training condition.
>
> @danpovey <https://github.com/danpovey> Do you agree ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/432/files#r48789340>.
>